app/views/pages/mailinglist/registration/step3.html.erb

```diff
@@ -1,4 +1,4 @@
-<section class="mailing-reg-main">
+<section class="mailing-reg-main" role="main"> 
```

Comment:
Could the role not be added at the template level?

Reply:
not really, need to be able to control when its needed and when it isnt